src/ui/marker.js

```diff
@@ -282,7 +283,7 @@ export default class Marker extends Evented {
     remove() {
         if (this._map) {
             this._map.off('click', this._onMapClick);
-            this._map.off('move', this._update);
+            this._map.off('move', () => this._update(true));
```

Comment:
This won't actually remove the listener that was added previously, because this is a different function. You need to pass the exact reference — e.g. by creating a separate `_updateMoving` that internally calls `_update(true)`.

Reply:
Done!